### PR TITLE
1.0.0/home screen component

### PR DESCRIPTION
**Pull request by FathimaNifa**
Removed second card and added parameters for the HomeScreen Component and added text restriction for title.

### CODE REVIEW

File: app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.nifa.fuel_buddy.R
+
+
+@Composable
+fun HomeScreenComponent(modifier: Modifier = Modifier,
+                        painterId: Int,
```
Comment (KannanGOfficial) [inline app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt]: Name it as imageDrawable Id

File: app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.nifa.fuel_buddy.R
+
+
+@Composable
+fun HomeScreenComponent(modifier: Modifier = Modifier,
```
Comment (KannanGOfficial) [inline app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt]: Name this function and file name as HomeScreenCard

File: app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                        ) {
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .background(colorResource(R.color.black)),
+        contentAlignment = Alignment.TopCenter
+    ) {
+
+
+        // First Card
```
Comment (KannanGOfficial) [inline app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt]: Remove this First card comment

File: app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+import com.nifa.fuel_buddy.R
+
+
+@Composable
+fun HomeScreenComponent(modifier: Modifier = Modifier,
+                        painterId: Int,
+                        title: String,
+                        distance: String
+                        ) {
+    Box(
```
Comment (KannanGOfficial) [inline app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt]: Remove this Box, we don't need here

File: app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.nifa.fuel_buddy.R
+
+
```
Comment (KannanGOfficial) [inline app/src/main/java/com/nifa/fuel_buddy/presentation/home/HomeScreenComponent.kt]: Need to format this file

File: .idea/deploymentTargetSelector.xml [hunk 6]
```diff
@@ -5,6 +5,9 @@
       <SelectionState runConfigName="app">
         <option name="selectionMode" value="DROPDOWN" />
       </SelectionState>
+      <SelectionState runConfigName="HomeScreenComponentPreview">
```
Comment (KannanGOfficial) [inline .idea/deploymentTargetSelector.xml]: Need to remove this file

File: .idea/gradle.xml [hunk 7]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```
Comment (KannanGOfficial) [inline .idea/gradle.xml]: Need to remove this file

**Comment by FathimaNifa** [comment]
I have completed all the changes and pushed the code

**Review by KannanGOfficial — APPROVED** [review]

**Pull request merged**